support-frontend/app/controllers/SubscriptionsController.scala

```diff
@@ -54,18 +54,22 @@ class SubscriptionsController(
     implicit val codec: com.gu.support.encoding.Codec[PriceCopy] = deriveCodec
   }
 
-  def pricingCopy(priceSummary: PriceSummary): PriceCopy =
+  def pricingCopy(priceSummary: PriceSummary): PriceCopy = {
+    val price = for {
```

Comment:
Maybe worth naming this to reflect the optionality and the fact that this is the discounted price? 

```suggestion
    val maybeDiscountedPrice = for {
```